runway/hooks/staticsite/auth_at_edge/templates/check_auth/__init__.py

```diff
@@ -0,0 +1,249 @@
+# pylint: skip-file
+"""Check the authorization information passed along via the cookie headers.
+
+When the information is not present or an error occurs due to verification a new
+request to Cognito will be made to authorize the user. The user will be taken
+to the Cognito login page to enter their information, and so long as it is valid,
+the information will be passed to the parsing agent.
+
+If a refresh token exists and it has expired (after 1 hour) do an automatic refresh
+of the credentials by redirecting the user to the refresh agent.
+"""
+import base64
+import datetime
+import hashlib
+import json
+import logging
+import random
+import re
+from urllib.parse import quote_plus, urlencode
+
+# pylint: disable=import-error
+from jose import jwt
+
+# pylint: disable=import-error
+from jwks_rsa.client import JwksClient
+# pylint: disable=import-error
```

Comment:
this should be in-line so it does not apply to the entire file, only a single line